lib/ldclient-rb/impl/datasystem/fdv1.rb

```diff
@@ -0,0 +1,227 @@
+require 'concurrent'
+require 'ldclient-rb/impl/datasystem'
+require 'ldclient-rb/impl/data_source'
+require 'ldclient-rb/impl/data_store'
+require 'ldclient-rb/impl/datasource/null_processor'
+require 'ldclient-rb/impl/flag_tracker'
+require 'ldclient-rb/impl/broadcaster'
+
+module LaunchDarkly
+  module Impl
+    module DataSystem
+      #
+      # FDv1 wires the existing v1 data source and store behavior behind the
+      # generic DataSystem surface.
+      #
+      # @private
```

Comment:
You know, I'm not sure if you actually need this `@private`. `lib/ldclient-rb/impl.rb` already includes a `@private` designation on the `Impl` namespace. Should be quick enough to test.